Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/DeviceIssuesPage/MiMDIssuesPage.tsx

```diff
@@ -39,27 +39,31 @@ function MiMDIssuesPage(props: { Meter: OpenXDA.Types.Meter }) {
     let dispatch = useAppDispatch();
 
     const [data, setData] = React.useState<SC.MiMDDailyStatistic[]>([]);
-    const [sortField, setSortField] = React.useState<keyof SC.MiMDDailyStatistic>('Date');
+    const [sortField, setSortField] = React.useState<keyof SC.MiMDDailyStatistic>('LastSuccessfulFileProcessed');
     const [ascending, setAscending] = React.useState<boolean>(false);
     const settings = useAppSelector(SystemCenterSettingSlice.Data);
     const settingStatus = useAppSelector(SystemCenterSettingSlice.Status);
 
     React.useEffect(() => {
         let handle = $.ajax({
             type: "GET",
-            url: `${homePath}api/SystemCenter/Statistics/MiMD/${props.Meter.AssetKey}`,
+            url: `${homePath}api/SystemCenter/Statistics/MiMD/${props.Meter.AssetKey}/${sortField}/${ascending ? '1' : '0'}`,
             contentType: "application/json; charset=utf-8",
             dataType: 'json',
             cache: false,
             async: true
-        }) as JQuery.jqXHR<SC.MiMDDailyStatistic[]>;
-        handle.done(d => setData(orderBy(d, [sortField], [ascending ? "asc" : "desc"])))
+        });
+        handle.done(d => setData(JSON.parse(d.toString())))
 
         return () => {
             if (handle.abort != undefined) handle.abort();
         }
     }, [props.Meter]);
```

Comment:
is there a reason we don't have `sortfield` and `ascending` in the dependencies instead of using the sepperate Effect for sorting?